common/yak/ssaapi/sf_result.go

```diff
@@ -61,17 +64,23 @@ func (r *SyntaxFlowResult) Name() string {
 	return ""
 }
 
-func (r *SyntaxFlowResult) GetAlertMsg(name string) (string, bool) {
+func (r *SyntaxFlowResult) GetAlertInfo(name string) (*sfvm.ExtraDescInfo, bool) {
 	if r == nil {
-		return "", false
+		return nil, false
 	}
 
 	if r.memResult != nil {
```

Comment:
use `r.rule` in all time, and set `r.rule` when create SyntaxFlowResult. 
in this pr modify function: `CreateResultFromQuery`